deployment/cdk/opensearch-service-migration/test/migration-console-stack.test.ts

```diff
@@ -57,13 +57,19 @@ test('Test that IAM policy does not contain fetch migration IAM statements when
     expect(iamPassRoleStatement).toBeFalsy()
 })
 
+
 test('Test that services yaml parameter is created', () => {
+    // Mock using local Dockerfile (which may not exist and would fail synthesis)
```

Comment:
After reviewing `network-stack.test.ts`, maybe when the mock was created too late in the lifecycle, in the other usages, the mock was declared before the tests.  Maybe you could shift these statement up so before objects are imported they are mocked out?

Reply:
I had tried it at the top level at first, and it was messing up the fetch stack image. But maybe I'll move it back to the top and see if I can figure out another way around that.